mitiq/pt/pt.py

```diff
@@ -51,38 +50,25 @@
 ]
 
 
-def execute_with_pauli_twirling(
+def tailor_circuit_with_pauli_twirling(
```

Comment:
Not opposed to this naming, but what do you think of `pauli_twirl_circuit`? Only reason I'm thinking to change it is because the main action (verb) of this function is twirl, not tailor.